test/smoke-fails/atomic-contended/Makefile

```diff
@@ -0,0 +1,64 @@
+include ../../Makefile.defs
+
+TESTNAME=atomic-contended
+TESTSRC_ALL=atomic-contended.cpp
+
+ifeq ($(TARGETS),)
+TARGETS =--offload-arch=$(AOMP_GPU)
+endif
+
+CC       =$(AOMP)/bin/clang++
+CFLAGS   =-O3 -std=c++11 -fopenmp $(TARGETS) -Wno-vla-cxx-extension
+
+RUNCMD   = ./performBuildRun.sh -a $(AOMP) -c $(CC) -f "$(CFLAGS)"             \
+                                -t $(TESTSRC_ALL)
+
+include ../Makefile.rules
+
+clean::
+	rm -f $(TESTNAME)_dpp    $(TESTNAME)_iter    $(TESTNAME)_none                \
+        $(TESTNAME)_dpp.ll $(TESTNAME)_iter.ll $(TESTNAME)_none.ll             \
+        $(TESTNAME)_dpp.s  $(TESTNAME)_iter.s  $(TESTNAME)_none.s
+
+# Atomic optimization kinds
```

Comment:
It worries me a bit that the demo compile commands and the actual test commands are separate between here and the bash script. I can see how this will lead to misunderstandings in the future.

I understand the rationale and I don't think it is a reason to not land this, but it still worries me a bit.